index.js

```diff
@@ -71,8 +71,39 @@ module.exports = url => {
 
     // Host all Web API handlers
     config.webApis.forEach(
-        ({ url, handler }) => {
-            app.get(
+        ({ url, handler, verb }) => {
+            app.methodToBeUsed = undefined;
+
+            switch (verb) {
+                case 'POST':
```

Comment:
Not that it handles everything, we still need to pass along POST data for a few verbs like this.